tests/framework/e2e/etcdctl.go

```diff
@@ -466,15 +466,41 @@ func (ctl *EtcdctlV3) AlarmDisarm(ctx context.Context, _ *clientv3.AlarmMember)
 }
 
 func (ctl *EtcdctlV3) AuthEnable(ctx context.Context) (*clientv3.AuthEnableResponse, error) {
-	var resp clientv3.AuthEnableResponse
-	err := ctl.spawnJsonCmd(ctx, &resp, "auth", "enable")
-	return &resp, err
+	args := []string{"auth", "enable", "-w", "json"}
+	cmd, err := SpawnCmd(append(ctl.cmdArgs(), args...), nil)
+	if err != nil {
+		return nil, err
+	}
+	err = cmd.Send(strings.Join(args, " "))
+	if err != nil {
+		return nil, err
+	}
+	defer cmd.Close()
+
+	_, err = cmd.ExpectWithContext(ctx, "Authentication Enabled")
+	if err != nil {
+		return nil, err
+	}
+	return &clientv3.AuthEnableResponse{}, nil
```

Comment:
Why do you modify methods `AuthEnable` and `AuthDisable`? 

Please only do one thing in one PR.

Reply:
Looks like existing implementations doesn't work at all.
Both methods return non-json responses `Authentication EnabledAuthentication Disabled` even if json output is requested. 
I should remove "-w", "json" from args though to be explicit.

Reply:
> Looks like existing implementations doesn't work at all.

Please show me the evidence. 

Again, one PR should only do one thing, and we always prefer to small PR instead of big one. please raise a separate PR if you think it's an issue and we can have more discussion in the PR.

Reply:
This is the right fix and I haven't yet created a PR to fix this. Go ahead to fix it.